tox.ini

```diff
@@ -17,8 +17,14 @@ install_command = pip install -U {opts} {packages}
 setenv = VIRTUAL_ENV={envdir}
 deps =  -r{toxinidir}/requirements.txt
         -r{toxinidir}/test-requirements.txt
+# If using command python -m testtools.run discover tests, it is possible
+# to break the column api service, because testtools will run all those tests
+# concurrently, yet column service is not supporting concurrent request now.
```

Comment:
...yet the column service does not support concurrent requests

Reply:
done